pkg/internal/mock/wallet/mock_wallet.go

```diff
@@ -18,16 +18,15 @@ type CloseableWallet struct {
 	CreateEncryptionKeyErr   error
 	CreateSigningKeyValue    string
 	CreateSigningKeyErr      error
+	AttachCryptoOperatorErr  error
+	ConvertKeyValue          []byte
+	ConvertKeyErr            error
```

Comment:
ConvertKey should not be part of the Crypto interface (it was a mistake to add key conversion in the Python implementation and they admit it)